src/Toolkit/templates/default/components/Grid.html.twig

```diff
@@ -0,0 +1,68 @@
+{%- props
+    xs = 0,
+    sm = 0,
+    md = 0,
+    lg = 0,
+    xl = 0,
+    align = '',
+    justify = '',
+    direction = '',
+    gap = 0,
+-%}
+
+{% set xsSize = false %}{%  if xs > 0 %}{% set xsSize = "true" %}{% endif %}
+{% set smSize = false %}{%  if sm > 0 %}{% set smSize = "true" %}{% endif %}
+{% set mdSize = false %}{%  if md > 0 %}{% set mdSize = "true" %}{% endif %}
+{% set lgSize = false %}{%  if lg > 0 %}{% set lgSize = "true" %}{% endif %}
+{% set xlSize = false %}{%  if xl > 0 %}{% set xlSize = "true" %}{% endif %}
+
+{% set alignClass = '' %}
+{% if align == 'start' %}{% set alignClass = 'items-start' %}
+{% elseif align == 'center' %}{% set alignClass = 'items-center' %}
+{% elseif align == 'end' %}{% set alignClass = 'items-end' %}
+{% endif %}
+
+{% set justifyClass = '' %}
+{% if justify == 'start' %}{% set justifyClass = 'justify-start' %}
+{% elseif justify == 'center' %}{% set justifyClass = 'justify-center' %}
+{% elseif justify == 'end' %}{% set justifyClass = 'justify-end' %}
+{% elseif justify == 'between' %}{% set justifyClass = 'justify-between' %}
+{% elseif justify == 'around' %}{% set justifyClass = 'justify-around' %}
+{% elseif justify == 'evenly' %}{% set justifyClass = 'justify-evenly' %}
+{% endif %}
+
+{% set directionClass = '' %}
+{% if direction == 'row' %}{% set directionClass = 'flex-row' %}
+{% elseif direction == 'column' %}{% set directionClass = 'flex-col' %}
+{% endif %}
+
+{% set gapClass = '' %}
+{% if gap > 0 %}
+    {% set gapClass = 'gap-' ~ gap %}
+{% endif %}
+{%- set style = html_cva(
+    base: 'grid',
+    variants: {
+        xsSize: {
+            "true": 'grid-cols-' ~ xs,
+        },
+        smSize: {
+            "true": 'sm:grid-cols-' ~ sm,
+        },
+        mdSize: {
+            "true": 'md:grid-cols-' ~ md,
+        },
+        lgSize: {
+            "true": 'lg:grid-cols-' ~ lg,
+        },
+        xlSize: {
+            "true": 'xl:grid-cols-' ~ xl,
+        },
+    },
+) -%}
```

Comment:
That's a lot of code, could it be optimized?

Also, having dynamic classes like `'grid-cols-' ~ xs` will requires:
- users with Tailwind 3 to explicitly configure [safe-list classes](https://v3.tailwindcss.com/docs/content-configuration#safelisting-classes) 
- users with Tailwind 4 to... **not be able to use this component**, since [dynamic classes](https://tailwindcss.com/docs/detecting-classes-in-source-files#dynamic-class-names) and [safe-list classes has been removed in Tailwind 4](https://tailwindcss.com/docs/upgrade-guide#using-a-javascript-config-file) 😬

Reply:
Real question, do you think we should really provide a `Grid` component? Does it make really sense?

Not providing it would make everyone's life easier IMO